src/index.scss

```diff
@@ -67,6 +67,7 @@
 @import "Pagination/";
 @import "Select/";
 @import "SeparatorList/";
+@import "Sidebar/";
```

Comment:
I forgot, we also need to add an export to `src/index.js` to make the component available to consumers of `@narmi/design_system`

Reply:
Added this!